charts/oes/values.yaml

```diff
@@ -203,6 +213,11 @@ gate:
     ## Reach out over support@opsmx.com for pre-configured user credentials
     fileBasedAuthentication:
       enabled: false
+    oesui:
+      externalUrl: ^https?://(?:localhost|OES_UI_LOADBALANCER_IP|opsmx.com)(?::[1-9]\d*)?/?
+    server:
```

Comment:
Let's add comment on when a user can reconfigure this timeout, how it makes a difference.

Reply:
Done